--- a/compiler/rustc_target/src/spec/mod.rs
+++ b/compiler/rustc_target/src/spec/mod.rs
@@ -1770,7 +1770,9 @@
     ("x86_64-unknown-linux-ohos", x86_64_unknown_linux_ohos),

     ("x86_64-unknown-linux-none", x86_64_unknown_linux_none),

+    ("x86_64-unknown-toyos", x86_64_unknown_toyos),
+
     ("thumbv6m-nuttx-eabi", thumbv6m_nuttx_eabi),
     ("thumbv7a-nuttx-eabi", thumbv7a_nuttx_eabi),
     ("thumbv7a-nuttx-eabihf", thumbv7a_nuttx_eabihf),
