--- a/library/std/src/sys/thread_local/mod.rs
+++ b/library/std/src/sys/thread_local/mod.rs
@@ -29,7 +29,8 @@
         target_os = "uefi",
         target_os = "zkvm",
         target_os = "trusty",
         target_os = "vexos",
+        target_os = "toyos",
     ) => {
         mod no_threads;
         pub use no_threads::{EagerStorage, LazyStorage, thread_local_inner};
@@ -100,6 +101,7 @@
             target_os = "zkvm",
             target_os = "trusty",
             target_os = "vexos",
+            target_os = "toyos",
         ) => {
             pub(crate) fn enable() {
                 // FIXME: Right now there is no concept of "thread exit" on
